### PR TITLE
W 15793257.separate rollup schedule

### PR DESCRIPTION
**Pull request by peterzxu-crm**

### CODE REVIEW

File: carbonj.service/src/main/java/com/demandware/carbonj/service/db/points/DataPointStagingStore.java [hunk 1]
```diff
@@ -86,18 +92,20 @@ public DataPointStagingStore(MetricRegistry metricRegistry, StagingFiles staging
...
-        this.s = Executors.newSingleThreadScheduledExecutor();
+        ScheduledExecutorService s = Executors.newSingleThreadScheduledExecutor();
         this.queue = new ArrayBlockingQueue<>( queueSize );
         IntervalProcessorTaskFactory taskFactory = new IntervalProcessorTaskFactoryImpl( pointStore);
         this.intervalProcessors = new IntervalProcessors(metricRegistry, stagingIntervalQueueConsumerBatchSize,
             stagingIntervalsQueueSizePerDb, stagingIntervalsQueueConsumersPerDb, taskFactory, emptyQueuePauseInMillis );
-        this.s.scheduleWithFixedDelay(this::propagate, 1, timeAggrJobIntervalInMins, TimeUnit.MINUTES );
-        this.s.scheduleWithFixedDelay(this::cleanup, 15, 5, TimeUnit.MINUTES );
+        s.scheduleAtFixedRate(this::propagateDb5m7d, 1, timeAggrJobIntervalInMins, TimeUnit.MINUTES );
+        s.scheduleAtFixedRate(this::propagateDb30m2y, 1 + timeAggrJobIntervalInMins / 2, timeAggrJobIntervalInMins, TimeUnit.MINUTES );
```
Comment (sholavanalli) [inline carbonj.service/src/main/java/com/demandware/carbonj/service/db/points/DataPointStagingStore.java:101]: The 2 jobs will collide every 4 hrs.

File: carbonj.service/src/main/java/com/demandware/carbonj/service/db/points/StagingFileSetCollector.java [hunk 2]
```diff
@@ -19,34 +20,34 @@
...
-                                    SortedStagingFile sortedFile = f.sort(lastSorted);
-                                    log.debug("sorted file: [" + sortedFile + "]");
+                                    log.info( "sorting ..." );
+                                    SortedStagingFile sortedFile = f.sort(lastSorted, dbName);
+                                    log.info("sorted file: [" + sortedFile + "]");
                                     sortedFiles.add( sortedFile );
                                 }
                 );
-        log.debug("sorted files: [" + sortedFiles + "]");
+        log.info("sorted files: [" + sortedFiles + "]");
```
Comment (sholavanalli) [inline carbonj.service/src/main/java/com/demandware/carbonj/service/db/points/StagingFileSetCollector.java:50]: Change these back to debug. Logs will be very verbose.

File: carbonj.service/src/main/java/com/demandware/carbonj/service/db/points/SystemSort.java [hunk 3]
```diff
@@ -130,7 +130,7 @@ private void addTmpDirArg( List<String> args )
     @Override
     public void sort( File inFile, Optional<File> extraMergeFile, File outFile) throws IOException
     {
-        log.debug(String.format("sorting [%s], with mergeFile [%s], into [%s]", inFile, extraMergeFile.orElse( null ), outFile));
+        log.info(String.format("sorting [%s], with mergeFile [%s], into [%s]", inFile, extraMergeFile.orElse( null ), outFile));
```
Comment (sholavanalli) [inline carbonj.service/src/main/java/com/demandware/carbonj/service/db/points/SystemSort.java:133]: Change back to debug?

File: carbonj.service/src/main/java/com/demandware/carbonj/service/db/points/StagingFileSetCollector.java (hunk 2 above)
Comment (peterzxu-crm) [reply carbonj.service/src/main/java/com/demandware/carbonj/service/db/points/StagingFileSetCollector.java]: done.

File: carbonj.service/src/main/java/com/demandware/carbonj/service/db/points/SystemSort.java (hunk 3 above)
Comment (peterzxu-crm) [reply carbonj.service/src/main/java/com/demandware/carbonj/service/db/points/SystemSort.java]: done.

**Pull request merged**